systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlIsolatedST.java

```diff
@@ -101,14 +102,14 @@ void testCruiseControlWithRebalanceResourceAndRefreshAnnotation(ExtensionContext
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(clusterName, 3, 3).build());
         resourceManager.createResource(extensionContext, KafkaRebalanceTemplates.kafkaRebalance(clusterName).build());
 
-        KafkaRebalanceUtils.doRebalancingProcess(clusterName);
+        KafkaRebalanceUtils.doRebalancingProcess(new Reconciliation("test", "kind", "ns", "name"), clusterName);
```

Comment:
You seem to use sometimes `ns` and sometimes `namespace`. Consistency might make things easier in the future. Also, these are STs, so this probably runs somewhere? Should it use areal names for resource, kind, namespace etc.?